websites/P/Physics Wallah/presence.ts

```diff
@@ -0,0 +1,148 @@
+const presence = new Presence({
+		clientId: "1134044987277975616",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let mediaTimestamps: [number, number];
+
+const enum Assets {
+	Paused = "https://i.imgur.com/nR0HSqz.png",
+	HomePage = "https://i.imgur.com/pIrO5z2.png",
+	Studying = "https://i.imgur.com/2ZFNWje.png",
+	Playing = "https://i.imgur.com/7tDTtPC.png",
+	Scrolling = "https://i.imgur.com/klh7wd3.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/OxQKsGm.png",
+			startTimestamp: browsingTimestamp,
+		},
+		Path = document.location.pathname;
```

Comment:
Destructure the multiple location properties of `document.location` and `window.location`
```suggestion
		{ pathname, href, search } = document.location;
```